interface/patient_file/reminder/patient_reminders.php

```diff
@@ -28,6 +28,9 @@
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
 
 <style type="text/css">@import url(../../../library/dynarch_calendar.css);</style>
+
+<!--Adding Jquery Plugin-->
+<script type="text/javascript" src="../../../library/js/jquery.1.3.2.js"></script>
```

Comment:
$GLOBALS['srcdir'] or $srcdir as found in globals.inc instead of the relative path.  even better, move a copy of the library over to the assets/js directory, as it is not a libreEHR library, but rather an externally maintained resource.